src/test/test_conformance.cpp

```diff
@@ -219,11 +227,17 @@ TEST_CONFORMANCE("rsh64-reg-neg.data")
 TEST_CONFORMANCE("sdiv32-by-zero-imm.data")
 TEST_CONFORMANCE("sdiv32-by-zero-reg.data")
 TEST_CONFORMANCE("sdiv32-imm.data")
+TEST_CONFORMANCE("sdiv32-intmin-by-negone-imm.data")
+TEST_CONFORMANCE("sdiv32-intmin-by-negone-reg.data")
```

Comment:
Does not seem important and if it were, they would be added in the bpf_conformance repository where these .data files are maintained rather than here.